lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -1,24 +1,94 @@
-import * as React from 'react';
-import { DatePicker } from '../DatePicker';
+import React, { useState } from 'react';
```

Comment:
Per https://github.com/mui-org/material-ui-pickers/issues/1631

```suggestion
import * as React from 'react';
```

Reply:
#1631 shows;
`Replace usage of import React, { useState } from 'react'; with import React from 'react'; React.useState.`

Is the wildcard * the preferred syntax?

Reply:
Thanks for raising, this standard originates from https://github.com/mui-org/material-ui/pull/19802. I have updated the issue.